setup.py

```diff
@@ -46,11 +46,13 @@ def verify_gpl_dependency():
         os.environ["SLUGIFY_USES_TEXT_UNIDECODE"] = "yes"
 
     if not os.getenv("AIRFLOW_GPL_UNIDECODE") and not os.getenv("SLUGIFY_USES_TEXT_UNIDECODE") == "yes":
-        raise RuntimeError("By default one of Airflow's dependencies installs a GPL "
-                           "dependency (unidecode). To avoid this dependency set "
-                           "SLUGIFY_USES_TEXT_UNIDECODE=yes in your environment when you "
-                           "install or upgrade Airflow. To force installing the GPL "
-                           "version set AIRFLOW_GPL_UNIDECODE")
+        raise RuntimeError(
+            "By default, one of Airflow's dependencies (unidecode) is GPL licensed .\n"
+            "In order to proceed with installation, "
+            "you will need to set one of the following environment variables:\n"
+            "To disallow the dependency, export SLUGIFY_USES_TEXT_UNIDECODE=yes.\n"
```

Comment:
This sentiment ("disallow") isn't quite right. What this env var is doing is letting the installer choose between a GPL-licensed dep, or a MIT licensed version instead.

Something like "to use a non-GPL licensed replacment" might be more accurate.

Reply:
Perhaps these environment variables should be rolled into a single variable, such as `allow_gpl_dependency` (yes or no).

Reply:
Sadly because of what consumes this (the setup.py of a dep two levels down) and the way pip/setup.py runs each dep in its own process this is not possible.

For 2.0.0 we will hopefully remove the need for this entirely by using a different module

Reply:
Cool. By the way, what is the issue with using a GPL'd dependency? To my understanding, there is no hard-linking with this module that would require downstream projects to be open source.

Reply:
That was not the view of the Apache legal team. I don't know the exact details, but they (view our project mentors) requested this feature.